src/cli.ts

```diff
@@ -75,8 +75,22 @@ class OdoChannelImpl implements OdoChannel {
         return jsonData;
     }
 
+    getToken(value: string) {
+        const regex = /--token\s*=\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ?  value.replace(tokenRegex[1], '**********') : value;
+    }
+
+    getCredentialsLogin(value: string) {
+        const regex = /-p\s*(.\w[a-zA-Z0-9!@#$%^&*()_+\-=\[\]{};':"\\|,.<>\/?]*)/;
+        const tokenRegex = value.match(regex);
+        return (tokenRegex) ? value.replace(tokenRegex[0], '-p **********'): value;
+    }
+
     print(text: string): void {
-        const textData = this.prettifyJson(text);
+        let textData = this.prettifyJson(text);
+        textData = this.getToken(textData);
+        textData = this.getCredentialsLogin(textData);
```

Comment:
Can we merge these 3 lines and make use of one field. It seems that it is using the recursive method to determine the final value of `textData`.